assemblies/features/enterprise-legacy/pom.xml

```diff
@@ -31,9 +31,11 @@
     <groupId>org.apache.karaf.features</groupId>
     <artifactId>enterprise-legacy</artifactId>
     <packaging>pom</packaging>
+    <version>${revision}</version>
     <name>Apache Karaf :: Assemblies :: Features :: Enterprise Legacy</name>
 
     <properties>
+        <revision>${enterprise-legacy.feature.tesb.version}</revision>
```

Comment:
Declared property is `enterprise-legacy.features.tesb.version` (with an `s`)

Reply:
typo corrected.